openpype/hosts/nuke/api/lib.py

```diff
@@ -1057,12 +1057,19 @@ def add_deadline_tab(node):
     node.addKnob(knob)
 
     knob = nuke.Int_Knob("deadlineConcurrentTasks", "Concurrent tasks")
+    #  zero as default will trigger value from Setting during collection
+    # look to precollect_write.py
```

Comment:
```suggestion
    # zero as default will get value from Settings during collection
    # instead of being an explicit user override, see precollect_write.py
```